src/plugins/automodContext/index.tsx

```diff
@@ -0,0 +1,80 @@
+/*
+ * Vencord, a Discord client mod
+ * Copyright (c) 2024 Vendicated and contributors
+ * SPDX-License-Identifier: GPL-3.0-or-later
+ */
+
+import { disableStyle, enableStyle } from "@api/Styles";
+import ErrorBoundary from "@components/ErrorBoundary";
+import { Devs } from "@utils/constants";
+import definePlugin from "@utils/types";
+import { findByPropsLazy } from "@webpack";
+import { Button, NavigationRouter, SelectedGuildStore, Text } from "@webpack/common";
+const Kangaroo = findByPropsLazy("jumpToMessage");
+
+import style from "./style.css?managed";
```

Comment:
your styles don't alter things that don't belong to your plugin, so it doesn't need to be managed.

```suggestion
import "./style.css";
```